FlorenceBML/BMLacq38.xml

```diff
@@ -34,6 +35,376 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="81" to="88"/>
+                                        </item>
+                                        <item xml:id="q13" n="13">
+                                            <dim unit="leaf">5</dim>
+                                            <locus from="89" to="93"/>
+                                            s.l. 1, no stub
+                                        </item>
+                                        <item xml:id="q14" n="14">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="94" to="101"/>
+                                        </item>
+                                        <item xml:id="q15" n="15">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="102" to="109"/>
+                                        </item>
+                                        <item xml:id="q16" n="16">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="110" to="117"/>
+                                        </item>
+                                        <item xml:id="q17" n="17">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="118" to="125"/>
+                                        </item>
+                                        <item xml:id="q18" n="18">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="126" to="133"/>
+                                        </item>
+                                        <item xml:id="q19" n="19">
+                                            <dim unit="leaf">9</dim>
+                                            <locus from="134" to="142"/>
+                                            3, 6, 9 s. l., stubs before 1, 4 and 7 
+                                        </item>
+                                        <item xml:id="q20" n="20">
+                                            <dim unit="leaf">3</dim>
+                                            <locus from="143" to="145"/>
+                                            1 s.l., stub after 3
+                                        </item>
+                                        <item xml:id="q21" n="21">
+                                            <dim unit="leaf">4</dim>
+                                            <locus from="v" to="viii"/>
+                                        </item>
+                                    </list>
+                                </collation>
+                                <condition key="intact">Rasures, corrections and rewrites for example on <locus target="#2r, #5v, #8r,
```

Comment:
I think "erasures" is the more common english term. It might also be better to move the description of textual corrections to an item in extras rather than have it under condition, which pertains more to the physical object.

Reply:
Done.